scripts/Tools/standard_script_setup.py

```diff
@@ -15,5 +16,5 @@
 import CIME.utils
 CIME.utils.check_minimum_python_version(2, 7)
 CIME.utils.stop_buffering_output()
-
-logging.basicConfig()
+# set up logging to file
+logging.basicConfig(level=logging.INFO)
```

Comment:
I moved some stuff that we had set to info into debug, the idea being that we want the default output level to be more informational than warn.